### PR TITLE
Hidden option to enforce maximum or minimum eth capability 

### PR DESCRIPTION
**Pull request by Gabriel-Trintinalia**
## PR description
Create hidden flags to enforce a range of Eth capabilities. With future versions of the protocol being implemented, we may need an easy way to compare capabilities without having to change the code every time.  Another use case would be to downgrade your node if an issue was found with a later protocol version.
 
This PR implements:

- Flag `--Xeth-capability-max` that enforces a maximum capability for the Eth protocol. Use: `--Xeth-capability-max=67`
- Flag `--Xeth-capability-min` that enforces a minimum capability for the Eth protocol. Use: `--Xeth-capability-min=65`

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if
    [updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

**Review by macfarla — COMMENTED** [review]
I think it makes sense - another use case would be to downgrade your node if an issue was found with a later protocol version, right?

File: besu/src/main/java/org/hyperledger/besu/cli/options/unstable/EthProtocolOptions.java [hunk 1]
```diff
@@ -34,6 +34,8 @@ public class EthProtocolOptions implements CLIOptions<EthProtocolConfiguration>
   private static final String LEGACY_ETH_64_FORK_ID_ENABLED =
       "--compatibility-eth64-forkid-enabled";
 
+  private static final String MAX_CAPABILITY_ENABLED = "--Xeth-max-capability-enabled";
```
Comment (macfarla) [inline besu/src/main/java/org/hyperledger/besu/cli/options/unstable/EthProtocolOptions.java:37]: re the name - generally the "enabled" suffix indicates a boolean 
what about
--X-eth-capability-max

Comment (Gabriel-Trintinalia) [reply besu/src/main/java/org/hyperledger/besu/cli/options/unstable/EthProtocolOptions.java]: Done

**Comment by Gabriel-Trintinalia** [comment]
> I think it makes sense - another use case would be to downgrade your node if an issue was found with a later protocol version, right?

That is another good example

File: ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/network/NetworkRunner.java [hunk 2]
```diff
@@ -78,6 +78,12 @@ public void start() {
       LOG.info("Starting Network.");
       setupHandlers();
       network.start();
+
+      LOG.info(
+          "Max supported protocol: {}",
```
Comment (macfarla) [inline ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/network/NetworkRunner.java:83]: this is logging all supported, not just max?

**Review by macfarla — APPROVED** [review]
LGTM

**Pull request merged**